site/content/integrate/apps/functionality/external-webhooks/_index.md

```diff
@@ -29,16 +29,17 @@ Using the above example values, the webhook URL will look like this:
 
 `https://my-mattermost-server/plugins/com.mattermost.apps/apps/my-app/webhook/my-sub-path?secret=cwsjhrdqebyf8qrnabtxio7k7r`
 
+When the external integration sends a `HTTP POST` request to this endpoint, your App will receive the request at the endpoint `/webhook/my-sub-path`. The same webhook URL can be used to process `HTTP HEAD` requests, in case the external integration requires this for webhook validation. The `/my-sub-path` part of the above URL is optional, and it can be defined as any string value.
```

Comment:
@hanzei Here's the snippet about `HEAD` requests

> The same webhook URL can be used to process `HTTP HEAD` requests, in case the external integration requires this for webhook validation.